corehq/apps/export/models/new.py

```diff
@@ -295,13 +292,12 @@ def get_rows(self, document, row_number):
             rows.append(ExportRow(data=row_data))
         return rows
 
-    def get_column(self, item_path, item_transform):
+    def get_column(self, item_path, column_transforms):
         # Columns should be unique by item path and item transform
+        # TODO: Actually, Column should be unique by item path and column.transforms minus any deid transforms?
```

Comment:
this is correct and was thinking about it the other day and didn't really come up with any good way to express it